kv/testlogger.go

```diff
@@ -0,0 +1,8 @@
+package kv
+
+type testLogger struct {
```

Comment:
This seems unused. Can we remove it?

Reply:
@pstibrany it's actually used by tests :) Same as the other one.